net/openssh/Makefile

```diff
@@ -82,7 +82,7 @@ endef
 
 define Package/openssh-server
 	$(call Package/openssh/Default)
-	DEPENDS+= +openssh-keygen
+	DEPENDS+= +openssh-keygen +libfido2
```

Comment:
I would change this to be en/disabled at compile time. Look at packages that use Config.in

Reply:
Pushed a fix for this. I guess I should I add this to the `openssh-server-pam` variant as well?

Reply:
If you want.